package-lock.json

```diff
@@ -1,10486 +0,0 @@
-{
```

Comment:
I'm not sure what this was used for, but there is no `package.json` for it...

Reply:
I think it accidently got committed when i ran npm i in the base of the repo!